SwiftGen.xcodeproj/project.pbxproj

```diff
@@ -576,6 +579,8 @@
 				09A87B4E1BCCA2C600D9B9F5 /* Resources */,
 				C40C2D80E9C1FEEAA4FD6B14 /* [CP] Embed Pods Frameworks */,
 				6C7EBDA1392A1BFCC9F0A2DA /* [CP] Copy Pods Resources */,
+				1CAEE405320C3DBD251A68B3 /* 📦 Embed Pods Frameworks */,
+				B8FA4CC6E874346645F002E3 /* 📦 Copy Pods Resources */,
```

Comment:
@djbe mine are also installed system-wide, that doesn't impact that :wink:

When you run `bundle install` bundler will install all the required gems, in the right versions, system-wide (unless told otherwise). Then you just need to run `bundle exec pod install` (instead of just `pod install`) to be sure to run the right version of your system-wide-installed CocoaPods, even if that one (the one defined in the Gemfile) isn't there last version installed in your system.